src/MinimalDelegation.sol

```diff
@@ -50,15 +53,26 @@ contract MinimalDelegation is IERC7821 {
     }
 
     function supportsExecutionMode(bytes32 mode) external pure override returns (bool result) {
-        return mode.isBatchedCall();
+        return mode.isBatchedCall() || mode.supportsOpData();
```

Comment:
i dont love this, doesn't seem like it will scale well

Reply:
yaa i agree but i think we basically have to do strict equality checks because i think its actually incorrect to proceed with a mode that has dirtied but unused bits